api/controllers/reviewController.js

```diff
@@ -0,0 +1,36 @@
+const { reviewService } = require("../services");
+const { catchAsync } = require("../utils/error");
+
+const createReview = catchAsync(async (req, res) => {
+  const { rating, comment, movieId } = req.body;
+  const accessToken = req.headers.authorization;
+  await reviewService.createReview(accessToken, rating, comment, movieId);
+  return res.status(200).json({ message: "CREATE REVIEW SUCCESS" });
+});
+
+const getReview = catchAsync(async (req, res) => {
+  const { movieId } = req.body;
```

Comment:
```suggestion
  const { movieId } = req.params;
```
해당 api가 실제로 하는 일은 review에 대한 data를 클라이언트입장에서 GET 할 수 있게 하는 것입니다. http method 또한 마찬가지로 이에 맞게 GET으로 수정됨과 동시에 매개변수를 body로 받는 것이 아닌, params로 받도록 수정되어야 겠습니다.